NEWS.md

```diff
@@ -1,3 +1,13 @@
+# DataFrames v1.0.x Patch Release Notes
+
+* patch v1.0.3: make sure `subset` checks if the passed condition function
+  returns a vector of values
```

Comment:
Update this if bumping to 1.1.

Reply:
updated and added more explanations. Fortunately the change we make will now throw an error so it will be easy to catch.